rubix/core/noise.py

```diff
@@ -29,12 +29,11 @@ def get_apply_noise(config: dict):
 
     logger = get_logger()
 
-    def apply_noise(inputs: dict[str, jax.Array]) -> dict[str, jax.Array]:
+    def apply_noise(rubixdata: object) -> object:
```

Comment:
I think I wrote that somewhere else, but instead of `object` we could create a type for `RubixData` or import the class here and explicitly add that to the typing argument

Reply:
Yes, this is now implemented on the documentation-sphinx branch